src/octree/types.rs

```diff
@@ -31,46 +31,59 @@ pub(in crate::octree) enum NodeChildrenArray<T: Default> {
 #[cfg_attr(feature = "serialization", derive(Serialize, Deserialize))]
 pub(in crate::octree) struct NodeChildren<T: Default> {
     /// The key value to signify "no child" at a given slot
-    pub(in crate::octree) default_key: T,
+    pub(in crate::octree) no_child_marker_key: T,
```

Comment:
@davids91 I am sorry, I can get rid of this, it sneaked itself in... I renamed it, during my dive through the code-base!